### ISSUE TITLE
News API

### ISSUE THREAD

**Issue by Xcom-Seller** [Qrome/marquee-scroller#251]
It looks like newsapi.org is no longer supported in the UK

**Comment by Qrome** [comment]
Is there an article or just from your personal experience?

**Comment by brianmiller1956** [comment]
I have 3 keys all obtained at different times since yesterday my news just shows null.
Not sure what has happened I will try and get a new key.

**Comment by Qrome** [comment]
Did you activate them?

**Comment by brianmiller1956** [comment]
Yes, these have all been working for over 18 months, they are all on different email adresses and we’re created at different times some months apart, I have just tried to get a new key and the response is that they are unable to serve the united  kingdom.
So it looks like they have shut the UK down, I have had no notification on any of the 3 addresses that this was happening. I have limited my calls to less than 30 calls per day as I know the limit is 100. It would be nice to have an explanation.

**Comment by Xcom-Seller** [comment]
I created a new key via a VPN based in the USA, that did not work either.
Seems it's just the UK they have stopped serving

**Comment by Qrome** [comment]
At the very bottom of the Terms page it says this:

"**Users in the United Kingdom**
We are unable to serve users in the United Kingdom. If you are based in the United Kingdom, you are not permitted to register for a News API account."

https://newsapi.org/terms

**Comment by Xcom-Seller** [comment]
Sorry I did not read that far down.
I am looking for a suitable replacement

**Comment by JayNaire** [comment]
> It looks like newsapi.org is no longer supported in the UK

Bummer! Maybe bbc news juicer may be an alternative; see [https://bbcnewslabs.co.uk/projects/juicer/](https://bbcnewslabs.co.uk/projects/juicer/) You have to email them for a key but it appears free for non-commercial use.

**Comment by brianmiller1956** [comment]
I did leave a message on the news-api support contact page but did not get a response. Just as a matter of interest are UK based news sources still available on news-api?

**Comment by Lofty73** [comment]
It seems they have stopped processing requests from UK based providers. The serial log upon startup 
Getting News Data for bbc-news
Getting News Data
http://newsapi.org/v2/top-headlines?sources=bbc-news&apiKey=a10a980d9fec4c4a82a0efd869475ab8
[HTTP] GET... code: 200
Start parsing...
status=ok
totalResults=10
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T16:37:28.4250943Z
url=http://www.bbc.co.uk/news/world-latin-america-62324354
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T16:37:24.2375138Z
url=http://www.bbc.co.uk/news/world-africa-62324294
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T14:22:23.0041522Z
url=http://www.bbc.co.uk/news/world-africa-62319112
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T13:37:21.2998966Z
url=http://www.bbc.co.uk/news/world-europe-62315781
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T11:07:18.8319074Z
url=http://www.bbc.co.uk/news/business-62318376
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T07:07:26.0053425Z
url=http://www.bbc.co.uk/news/world-asia-62316057
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T06:07:25.2547935Z
url=http://www.bbc.co.uk/news/world-australia-62314555
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T04:52:22.8468723Z
url=http://www.bbc.co.uk/news/world-us-canada-62314306
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-27T01:22:22.5812825Z
url=http://www.bbc.co.uk/news/world-us-canada-62157479
author=BBC News
Id=bbc-news
Name=BBC News
publishedAt=2022-07-26T23:22:23.207126Z
url=http://www.bbc.co.uk/news/world-us-canada-62310576
Version: 3.01
Sorry its long

**Comment by Qrome** [comment]
That is correct.   Newsapi.org is no longer allowed to provide news feeds to the UK.  This is mentioned on their Terms page.

**Comment by victor7376** [comment]
I tried emailing newsapi, but they seem to be ignoring emails.

I also looked at other providers and I cant see anyone else that supplies a json response in news.

**Comment by Qrome** [comment]
> I tried emailing newsapi, but they seem to be ignoring emails.
> 
> I also looked at other providers and I cant see anyone else that supplies a json response in news.

Did you look at this one?  https://newsdata.io/

**Comment by Xcom-Seller** [comment]
Just come back online a few minutes ago 11:30 GMT

**Comment by brianmiller1956** [comment]
Yes I can confirm news is back on, using existing key, hopefully for good. Thanks for the update

**Comment by brianmiller1956** [comment]
I have emailed them asking why we no longer have service in the UK, I will forward any response, if I get one.

**Comment by Xcom-Seller** [comment]
Mine is still working fine

**Comment by sorriso93** [comment]
same problem also from italy started in the last week
